src/util/prop-definition.js

```diff
@@ -59,7 +60,7 @@ export default class PropDefinition {
         case 'get':
         case 'serialize':
         case 'set':
-          if (typeof optVal === 'function') {
+          if (isFunction(optVal)) {
             this[option] = optVal;
           } else {
             console.error(option + ' must be a function.');
```

Comment:
I must have missed this in a previous review? I would prefer to leave out all console error calls directly. If we're going to start doing this (and I've thought it might be a good idea, anyways, I'd like to do something like:

```js
import { isUndefined } from './util/is-type';
import root from 'window-or-global';

function error (message) {
  const { DEBUG } = root;
  if (!isUndefined(DEBUG) && DEBUG) {
    throw new Error(message);
  }
}
```

Then just do:

```js
error(`${option} must be a function`);
```

You should also use template literals here instead of string concatenation.